pkg/apis/core/validation/validation.go

```diff
@@ -5616,19 +5616,18 @@ func ValidatePodResize(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 	}
 
 	// Do not allow removing resource requests/limits on resize.
-	if utilfeature.DefaultFeatureGate.Enabled(features.SidecarContainers) {
```

Comment:
Since we need to support an `emulated_version`, we need to keep this code for control plane. See KEP: https://github.com/kubernetes/enhancements/issues/4330 Let's still delete it from the kubelet code.

Reply:
done